src/config.ts

```diff
@@ -11,6 +11,14 @@ export const defaultConfig = {
   stega: {enabled: false},
 } satisfies ClientConfig
 
+/**
+ * Get the resource configuration, preferring the new `resource` property over the deprecated `~experimental_resource`
+ * @internal
+ */
+export function getResourceConfig(config: InitializedClientConfig) {
```

Comment:
This should either not be exported, or used throughout the code to normalize
```suggestion
function getResourceConfig(config: InitializedClientConfig) {
```

Reply:
I removed it all together and am just relying on the normalization and using config directly